Metzingen hinzufügen

pzi42 opened the pull request:
Bitte um Review meines Artikel zu Metzingen

fixes: #32

Commit:
Beginn einer Beschreibung der Stadt Metzingen

Commit:
Links korrigiert

Commit:
nochmalige Link Korrektur

Jannik-R reviewed (changes requested):
Ich würde den Eintrag übersichtlicher finden, wenn Überschriften verwendet würden, wie z.B. in der "Beispielort.md". 
Weiterhin währe eventuell noch ein Bild schön.

Commit:
Ueberschriften angelegt, Bilder eingefuegt.

Zur bessern Übersicht habe ich Ueberschriften eingefügt. Zu Illustration
habe ich noch Bilder eingefügt.

pzi42 commented:
Danke, ich denke du hast Recht, ein paar Überschriften tun gut und ein paar Bilder auch.

Jannik-R commented:
Wie sieht es bezüglich der Urheberrechte aus? 
Weil die Bilder jetzt direkt von der Seite der Stadt stammen.

pzi42 commented:
Über das Urheberrecht habe ich mir hier keine Gedanken gemacht.  Ich ging von einem internen - privaten Projekt zu Übungszwecken aus.

Ich werde die Bilder  gegen Aufnahmen von mir oder aus commons.wikimedia.org austauschen.
